content/docs/user-guide/networking/settings.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Networking And Multiplayer Settings
+description: A reference for console variables and other settings that can be used on clients and servers to configure networking and multiplayer.
+linktitle: Network-settings
+---
+
+## Overview
+This page documents console variables (CVars) and settings for AzNetworking and Multiplayer behaviour.
+
+## Client settings
+| Setting                | Description                                                                                                                                                                             | Default   | Notes |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|-------|
+| cl_clientport          | The port to bind to for game traffic when connecting to a remote host, a value of 0 will select any available port.                                                                     | 0         |       |
```

Comment:
It may be useful to add a quick note somewhere on cl_ sv_ bg_ prefixes somewhere as well.

Reply:
To me that should go on a the general cvar page: https://www.o3de.org/docs/user-guide/appendix/cvars/